src/CommunityToolkit.Maui/Views/Expander/Expander.shared.cs

```diff
@@ -84,27 +59,6 @@ public Expander()
 	/// </remarks>
 	public Action<TappedEventArgs>? HandleHeaderTapped { get; set; }
 
-	/// <inheritdoc />
-	public IView? Header
-	{
-		get => (IView?)GetValue(HeaderProperty);
-		set => SetValue(HeaderProperty, value);
-	}
-
-	/// <inheritdoc cref="ContentView.Content" />
-	public new IView? Content
-	{
-		get => (IView?)GetValue(Expander.ContentProperty);
-		set => SetValue(Expander.ContentProperty, value);
-	}
-
-	/// <inheritdoc />
-	public bool IsExpanded
-	{
-		get => (bool)GetValue(IsExpandedProperty);
-		set => SetValue(IsExpandedProperty, value);
-	}
-
 	/// <inheritdoc />
```

Comment:
Since `Direction` has a custom `set` method I don't use the SG to generate it. Maybe in the future, when we have `partial Property` we can come back here and improve it

Reply:
Isn't a good idea to cast all Enums to `int`, some enums don't have a int value or use the Flag attribute. Here I would say it's best to let devs handle that instead of forcing something

Reply:
sorry, accidentally deleted comment:
We can choose another more generic exception (or even create a custom) or just not cast to int. 
```
if (enum)
{
	public PropertyEnumType PropertyName
	{
		get => (PropertyEnumType)GetValue(PropertyNameProperty);
		set
		{
			if (!Enum.IsDefined(typeof(PropertyEnumType), value))
			{
				throw new InvalidEnumException(nameof(value), value, typeof(PropertyEnumType));
			}

			SetValue(PropertyNameProperty, value);
		}
	}
}
```

Or we can have partial method OnPropertyChanging(), where developer can add additional logic. It will be even more flexible:
public PropertyType PropertyName
	{
		get => (PropertyType)GetValue(PropertyNameProperty);
		set
		{
			OnPropertyChanging(value);
			SetValue(PropertyNameProperty, value);
		}
	}

Reply:
With that, we will infer a lot about enums used on SG, in this case I think it's better to leave at it's and measure that on our usage